content/docs/2.13/operate/events.md

```diff
@@ -28,3 +28,49 @@ KEDA emits the following [Kubernetes Events](https://kubernetes.io/docs/referenc
 | `TriggerAuthenticationDeleted`        | `Normal`  | When a TriggerAuthentication is deleted                                                                                     |
 | `ClusterTriggerAuthenticationAdded`   | `Normal`  | When a new ClusterTriggerAuthentication is added                                                                            |
 | `ClusterTriggerAuthenticationDeleted` | `Normal`  | When a ClusterTriggerAuthentication is deleted                                                                              |
+
+
+## CloudEvent Support (Experimental)
+
+### CloudEventSource Resource
+`CloudEventSource` resource now can be created in KEDA for emitting events to user's custom CloudEvent sink. Event will be emitted to both Kubernetes Events and CloudEvents Destination if CloudEventSource resource is created. This specification describes the `CloudEventSource` Custom Resource definition:
```

Comment:
> Event will be emitted to both Kubernetes Events and CloudEvents Destination if CloudEventSource resource is created.

I personally think we should not rely on this CRD to decide if we emit these events to Kubernetes Events, can we please change the implementation for this?